### ISSUE TITLE
JaQu CLOB single-quoting bug

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sagarswathi/h2database#277]
```
I'm experimenting with JaQu and I think I've found a bug for which I've 
attached a Test Case.

Since I don't know how to define a CLOB column with JaQu, I manually create the 
table.
```

Original issue reported on code.google.com by `James.Mo...@gmail.com` on 19 Jan 2011 at 3:40

**Comment by GoogleCodeExporter** [comment]
```
I found the bug, it is now fixed in the trunk.

Thanks a lot! I will also add your test case.
```

Original comment by `thomas.t...@gmail.com` on 25 Jan 2011 at 7:29
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 1.3.150
```

Original comment by `thomas.t...@gmail.com` on 28 Jan 2011 at 7:23
- Changed state: **Fixed**
